src/test/java/com/google/javascript/cl2dts/DeclarationGeneratorTests.java

```diff
@@ -67,7 +67,7 @@ static File getGoldenFile(final File input) {
   static String getTestFileText(final File input) throws IOException {
     String text = Files.asCharSource(input, Charsets.UTF_8).read();
     // Strip test comments in .d.ts golden files starting with '//!!'.
-    return text.replaceAll("^\\s*//!!.*\\n", "");
+    return text.replaceAll("[ ]*//!!.*\\n", "");
```

Comment:
Problem was missing multi-line and `\n` being part of `\s`. Didn't know about `(?m)` that fixes it nicely.